weave-js/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": false,
     "isolatedModules": true,
-    "jsx": "preserve",
+    "jsx": "react",
```

Comment:
This setting is already used in the frontend
https://github.com/wandb/core/blame/master/frontends/app/tsconfig.json#L15